enterprise_gateway/services/kernelspecs/handlers.py

```diff
@@ -95,10 +93,8 @@ async def get(self, kernel_name):
         d = apply_user_filter(model, kernel_user)
 
         if d is None:
-            self.log.debu("You shouldnt been doing that")
-            # User is not Authorized to know kernel
-            # specs
-            # Do something
+            raise web.HTTPError(403, u'User %s is not authorized to use this kernel spec %s'
```

Comment:
I think we should drop 'this' or add a separator like "kernel spec: %s" that "introduces the variable kernel spec name.

Reply:
Changed to `User %s is not authorized to use kernel spec %s`